### DISCUSSION TITLE
Websocket connection failed

### DISCUSSION THREAD

**Discussion by rosiehollidaydev** [supabase/supabase#1287] [Questions] [answered]
Hey all, been working on my app for a couple of weeks with supabase as a back end.

Today I am getting a websocket connection failed error with no changes made since it was working yesterday...

Any advice would be appreciated

**Comment by inian** [comment, marked as answer]
Hey @rosiemcposie, can you email beta@supabase.io with your project details? Thanks!

**Reply by rosiehollidaydev** [reply]
Can do.

**Reply by rosiehollidaydev** [reply]
Amazing support!
